arrow/util.py

```diff
@@ -9,6 +10,18 @@
 from arrow.constants import MAX_TIMESTAMP, MAX_TIMESTAMP_MS, MAX_TIMESTAMP_US
 
 
+def seconds_in_month(dt):
+
+    year = dt.year
+    month = dt.month
+
+    days_in_month = calendar.monthrange(year, month)[1]
+
+    seconds = days_in_month * 86400
```

Comment:
We should probably get 86400 into a variable rather than having a magic constant.

Reply:
Yeah to be honest I think this function has some problems. Because we are calculating total seconds for the current month that could mean we under/over estimate the delta for the follwing month

Reply:
Yeah I'm thinking there's a few issues with this method.